hive/r/2-hive-queries.R

```diff
@@ -0,0 +1,16 @@
+
+source("r/0-configure.R")
+
+# queries -----------------------------------------------------------------
+
+# parameters
+group_definition <- "substring(pickup_datetime, 12, 2)"
+group_name <- "hour"
+
+# count by weekday on the taxi dataset
+query_group_by <- sprintf("SELECT %s AS %s, COUNT(*) AS count FROM %s GROUP BY %s",
+                          group_definition, group_name, name_table, group_definition)
+cat(query_group_by, ";", sep = "",
+    file = file.path(folder_hql, "2-count-by-weekday.sql"))
+count_hour <- rhive.query(query_group_by)
+View(count_hour)
```

Comment:
Never use `View()` in scripts. This opens additional windows, which is just a bad side-effect to have in a script.  Use `head` or `print()` or similar to view the results from a script.

For interactive use, `View()` is fine.

Please change.